packages/vue/src/components/SynVueButtonGroup.vue

```diff
@@ -64,3 +64,7 @@ defineEmits<{
     <slot />
   </syn-button-group>
 </template>
+
+<script lang="ts">
```

Comment:
This should not be on this position. If it is empty, it should be not there at all.